cheatsheets/C-Based_Toolchain_Hardening_Cheat_Sheet.md

```diff
@@ -644,10 +644,10 @@ There is one caveat with using some of the guards: Apple only provides them for
 
 #### Windows
 
-Visual Studio offers a number of debugging aides for use during development. The aides are called [Managed Debugging Assistants (MDAs)](https://docs.microsoft.com/en-us/dotnet/framework/debug-trace-profile/diagnosing-errors-with-managed-debugging-assistants). You can find the MDAs on the _Debug_ menu, then _Exceptions_ submenu. MDAs allow you to tune your debugging experience by, for example, filter exceptions for which the debugger should snap. For more details, see Stephen Toub's _[Let The CLR Find Bugs For You With Managed Debugging Assistants](https://docs.microsoft.com/en-us/archive/msdn-magazine/2006/may/let-the-clr-find-bugs-for-you-with-managed-debugging-assistants)_.
+Visual Studio offers a number of debugging aids for use during development. The aids are called [Managed Debugging Assistants (MDAs)](https://docs.microsoft.com/en-us/dotnet/framework/debug-trace-profile/diagnosing-errors-with-managed-debugging-assistants). You can find the MDAs on the _Debug_ menu, then _Exceptions_ submenu. MDAs allow you to tune your debugging experience by, for example, filtering exceptions for which the debugger should snap. For more details, see Stephen Toub's _[Let The CLR Find Bugs For You With Managed Debugging Assistants](https://docs.microsoft.com/en-us/archive/msdn-magazine/2006/may/let-the-clr-find-bugs-for-you-with-managed-debugging-assistants)_.
```

Comment:
I think 'aide' is the correct work here. From the American Heritage Dictionary, used as a noun:

- **aide**: An assistant; a helper. Example: _"A nurse's aide"._
- **aid**: The act or result of helping; assistance. Example: _"gave aid to the enemy"._

I think aide is a better fit here.

Reply:
The reason I changed it was an aide is a person who helps someone. The helpers in this context aren't human beings, which is why "aid" is the correct word here. [https://www.dictionary.com/e/aid-vs-aide/](url)

> Additionally, the word aide only refers to people, while aid can refer to people or objects.